### PR TITLE
[DDW-1222] Fix theming issues

### PR DESCRIPTION
**Pull request by DeeJayElly**
This PR fixes all Daedalus UI issues stated in the document https://docs.google.com/document/d/1VuXLJ-IysCXUuMUkEHFABCCKAMo0OCJEzVIwLoCbKgs/edit?ts=5e7c7cc7

## Testing Checklist

- [Slack QA thread](https://input-output-rnd.slack.com/archives/GGKFXSKC6/p1585337248037000)
- [ ] Test if all the UI issues from document https://docs.google.com/document/d/1VuXLJ-IysCXUuMUkEHFABCCKAMo0OCJEzVIwLoCbKgs/edit?ts=5e7c7cc7 are resolved

---

### Basics

- [ ] PR has been assigned and has appropriate labels (`feature`/`bug`/`chore`, `release-x.x.x`)
- [ ] PR is updated to the most recent version of the target branch (and there are no conflicts)
- [ ] PR has a good description that summarizes all changes and shows some screenshots or animated GIFs of important UI changes
- [ ] CHANGELOG entry has been added to the top of the appropriate section (*Features*, *Fixes*, *Chores*) and is linked to the correct PR on GitHub
- [ ] Automated tests: All acceptance and unit tests are passing (`yarn test`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *development* build (`yarn dev`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *production* build (`yarn package` / CI builds)
- [ ] There are no *flow* errors or warnings (`yarn flow:test`)
- [ ] There are no *lint* errors or warnings (`yarn lint`)
- [ ] There are no *prettier* errors or warnings (`yarn prettier:check`)
- [ ] There are no missing translations (running `yarn manage:translations` produces no changes)
- [ ] Text changes are proofread and approved (Jane Wild / Amy Reeve)
- [ ] Japanese text changes are proofread and approved (Junko Oda)
- [ ] UI changes look good in all themes (Alexander Rukin)
- [ ] Storybook works and no stories are broken (`yarn storybook`)
- [ ] In case of dependency changes `yarn.lock` file is updated

### Code Quality
- [ ] Important parts of the code are properly commented and documented
- [ ] Code is properly typed with flow
- [ ] React components are split-up enough to avoid unnecessary re-renderings
- [ ] Any code that only works in main process is neatly separated from components

### Testing
- [ ] New feature/change is covered by acceptance tests
- [ ] New feature/change is manually tested and approved by QA team
- [ ] All existing acceptance tests are still up-to-date
- [ ] New feature/change is covered by Daedalus Testing scenario
- [ ] All existing Daedalus Testing scenarios are still up-to-date

### After Review
- [ ] Merge the PR
- [ ] Delete the source branch
- [ ] Move the ticket to `done` column on the YouTrack board
- [ ] Update Slack QA thread by marking it with a green checkmark

### CODE REVIEW

**Comment by gnpf** [comment]
@DeeJayElly I'm not sure if the issues about the spinners are resolved. @a-rukin can you please advice on this? Thanks!

BG3AKk8AUd
VO7HUuBqUm

File: source/renderer/app/components/splash/SplashNetworkITN.js [hunk 1]
```diff
@@ -47,16 +48,18 @@ const messages = defineMessages({
 type Props = {
   onClose: Function,
   openExternalLink: Function,
+  currentTheme: string,
 };
 
+@inject('stores')
```
Comment (nikolaglumac) [inline source/renderer/app/components/splash/SplashNetworkITN.js]: @DeeJayElly please remove this - it is only allowed to inject stores into containers and then pass the values to the components.

File: source/renderer/app/components/splash/SplashNetworkITN.js [hunk 2]
```diff
@@ -76,6 +79,7 @@ export default class SplashNetworkITN extends Component<Props> {
         description={description}
         buttonLabel={buttonLabel}
         linkLabel={linkLabel}
+        currentTheme={currentTheme}
```
Comment (nikolaglumac) [inline source/renderer/app/components/splash/SplashNetworkITN.js]: @DeeJayElly why do we need to pass the currentTheme here?

File: source/renderer/app/components/staking/delegation-center/DelegationCenterBody.js [hunk 3]
```diff
@@ -16,7 +16,7 @@ const messages = defineMessages({
   },
   loadingStakePoolsMessage: {
     id: 'staking.delegationCenter.loadingStakePoolsMessage',
-    defaultMessage: '!!!Loading stake pools',
+    defaultMessage: '!!!Loading wallets',
```
Comment (nikolaglumac) [inline source/renderer/app/components/staking/delegation-center/DelegationCenterBody.js]: @DeeJayElly please revert this change

File: source/renderer/app/components/profile/terms-of-use/TermsOfUseText.scss [hunk 4]
```diff
@@ -4,7 +4,10 @@
   user-select: text;
 
   a {
-    color: var(--theme-support-settings-text-color);
+    border-bottom: 1px solid var(--rp-link-color);
```
Comment (nikolaglumac) [inline source/renderer/app/components/profile/terms-of-use/TermsOfUseText.scss]: @DeeJayElly please introduce new css var for this: `--theme-support-settings-link-color`

File: source/renderer/app/components/wallet/skins/AmountInputSkin.scss [hunk 5]
```diff
@@ -9,7 +9,7 @@
   font-size: 14px;
   position: absolute;
   right: 19px;
-  user-select: text;
+  user-select: none;
```
Comment (nikolaglumac) [inline source/renderer/app/components/wallet/skins/AmountInputSkin.scss]: @DeeJayElly why did we add this?

File: source/renderer/app/components/wallet/skins/AmountInputSkin.scss [hunk 6]
```diff
@@ -20,5 +20,5 @@
   position: absolute;
   right: 20px;
   text-transform: uppercase;
-  user-select: text;
+  user-select: none;
```
Comment (nikolaglumac) [inline source/renderer/app/components/wallet/skins/AmountInputSkin.scss]: @DeeJayElly why did we add this?

File: source/renderer/app/components/widgets/splash/Splash.js [hunk 7]
```diff
@@ -32,13 +34,14 @@ export default class SplashNetwork extends Component<Props> {
...
       linkLabel,
+      currentTheme,
       backgroundImage,
     } = this.props;
 
     return (
       <div className={styles.component}>
         <div className={styles.backgroundContainer}>
-          {backgroundImage && (
+          {currentTheme === THEMES.INCENTIVIZED_TESTNET && backgroundImage && (
```
Comment (nikolaglumac) [inline source/renderer/app/components/widgets/splash/Splash.js]: @DeeJayElly please add this check as a variable in the Profile store: `@observable isIncentivizedTestnetTheme = currentTheme === THEMES.INCENTIVIZED_TESTNET;`

File: source/renderer/app/components/wallet/skins/AmountInputSkin.scss (hunk 6 above)
Comment (DeeJayElly) [reply source/renderer/app/components/wallet/skins/AmountInputSkin.scss]: Text should be non selectable according to -> https://docs.google.com/document/d/1VuXLJ-IysCXUuMUkEHFABCCKAMo0OCJEzVIwLoCbKgs/edit?disco=AAAAJP3XFi0

File: source/renderer/app/components/wallet/skins/AmountInputSkin.scss (hunk 5 above)
Comment (DeeJayElly) [reply source/renderer/app/components/wallet/skins/AmountInputSkin.scss]: Text should be non selectable according to -> https://docs.google.com/document/d/1VuXLJ-IysCXUuMUkEHFABCCKAMo0OCJEzVIwLoCbKgs/edit?disco=AAAAJP3XFi0

File: source/renderer/app/components/wallet/skins/AmountInputSkin.scss (hunk 6 above)
Comment (nikolaglumac) [reply source/renderer/app/components/wallet/skins/AmountInputSkin.scss]: Got it - thanks!

File: source/renderer/app/components/wallet/skins/AmountInputSkin.scss (hunk 5 above)
Comment (nikolaglumac) [reply source/renderer/app/components/wallet/skins/AmountInputSkin.scss]: Cool!

File: source/renderer/app/components/staking/delegation-center/DelegationCenterBody.js (hunk 3 above)
Comment (DeeJayElly) [reply source/renderer/app/components/staking/delegation-center/DelegationCenterBody.js]: Reverted.

**Comment by a-rukin** [comment]
image
Are we not able to use "colored" spinners at all? 
I mean I face a similar issue in different cases

**Comment by nikolaglumac** [comment]
> image
> 
> Are we not able to use "colored" spinners at all? I mean I face a similar issue in different cases

@a-rukin no we need to implement a smart component for that...

File: source/renderer/app/components/splash/SplashNetworkITN.js (hunk 1 above)
Comment (DeeJayElly) [reply source/renderer/app/components/splash/SplashNetworkITN.js]: Its a leftover, i removed it.

**Comment by a-rukin** [comment]
image
is there any overlap? 
seems like there is something transparent under

File: source/renderer/app/components/splash/SplashNetworkITN.js (hunk 1 above)
Comment (DeeJayElly) [reply source/renderer/app/components/splash/SplashNetworkITN.js]: Its a leftover, i removed it.

File: source/renderer/app/components/splash/SplashNetworkITN.js (hunk 2 above)
Comment (DeeJayElly) [reply source/renderer/app/components/splash/SplashNetworkITN.js]: We need to recheck the current theme as the background for non incentivized themes should be plain without circles.

File: source/renderer/app/components/splash/SplashNetworkITN.js (hunk 1 above)
Comment (nikolaglumac) [reply source/renderer/app/components/splash/SplashNetworkITN.js]: Are you sure? I still see it on GH? 🤔

File: source/renderer/app/components/profile/terms-of-use/TermsOfUseText.scss (hunk 4 above)
Comment (DeeJayElly) [reply source/renderer/app/components/profile/terms-of-use/TermsOfUseText.scss]: It appears that variable already exists, so just added it instead --rp-link-color.

**Comment by DeeJayElly** [comment]
> image
> 
> is there any overlap? seems like there is something transparent under

There is some 'text-shadow: 0 -1px 0 rgba(0, 0, 0, 0.25)' on top of active item.

Should i remove it @a-rukin ?

File: source/renderer/app/components/splash/SplashNetworkITN.js [hunk 8]
```diff
@@ -1,6 +1,7 @@
 // @flow
 import React, { Component } from 'react';
 import { defineMessages, intlShape, FormattedHTMLMessage } from 'react-intl';
+import { inject } from 'mobx-react';
```
Comment (nikolaglumac) [inline source/renderer/app/components/splash/SplashNetworkITN.js]: @DeeJayElly we don't need this one...

File: source/renderer/app/components/splash/SplashNetworkITN.js [hunk 9]
```diff
@@ -47,6 +48,7 @@ const messages = defineMessages({
 type Props = {
   onClose: Function,
   openExternalLink: Function,
+  isIncentivizedTestnetTheme?: boolean,
```
Comment (nikolaglumac) [inline source/renderer/app/components/splash/SplashNetworkITN.js]: @DeeJayElly please make this mandatory prop 🙏

File: source/renderer/app/components/widgets/splash/Splash.js [hunk 10]
```diff
@@ -18,6 +18,7 @@ type Props = {
   description: Node,
   buttonLabel: string,
   linkLabel: string,
+  isIncentivizedTestnetTheme?: boolean,
```
Comment (nikolaglumac) [inline source/renderer/app/components/widgets/splash/Splash.js]: @DeeJayElly please make this one mandatory prop.

File: source/renderer/app/i18n/locales/defaultMessages.json [hunk 11]
```diff
@@ -2512,7 +2512,7 @@
         }
       },
       {
-        "defaultMessage": "!!!Loading stake pools",
+        "defaultMessage": "!!!Loading wallets",
```
Comment (nikolaglumac) [inline source/renderer/app/i18n/locales/defaultMessages.json]: @DeeJayElly please revert this change.

**Comment by a-rukin** [comment]
image
mini-switch border radius is too big - should be 2px

**Comment by a-rukin** [comment]
image

some padding is still here (less but still) -> need to remove it 
p.s. it can be wrong padding for switch

**Comment by a-rukin** [comment]
image
and 
image

different font weight

File: source/renderer/app/components/splash/SplashNetworkITN.js (hunk 8 above)
Comment (DeeJayElly) [reply source/renderer/app/components/splash/SplashNetworkITN.js]: Done

File: source/renderer/app/components/splash/SplashNetworkITN.js (hunk 9 above)
Comment (DeeJayElly) [reply source/renderer/app/components/splash/SplashNetworkITN.js]: Done

File: source/renderer/app/components/widgets/splash/Splash.js (hunk 10 above)
Comment (DeeJayElly) [reply source/renderer/app/components/widgets/splash/Splash.js]: It's optional for another Splash screen, <SplashNetworkFlight> doesn't need to pass it to <Splash> component.

Comment (DeeJayElly) [reply source/renderer/app/components/widgets/splash/Splash.js]: > It's optional for another Splash screen - <SplashNetworkFlight, doesn't need to pass it to component.

**Comment by a-rukin** [comment]
image
not fixed

File: source/renderer/app/components/widgets/splash/Splash.js (hunk 10 above)
Comment (nikolaglumac) [reply source/renderer/app/components/widgets/splash/Splash.js]: Yes so make it mandatory on the parent as well but not on the other splash screen.

File: source/renderer/app/i18n/locales/defaultMessages.json (hunk 11 above)
Comment (DeeJayElly) [reply source/renderer/app/i18n/locales/defaultMessages.json]: Done

File: source/renderer/app/components/widgets/splash/Splash.js (hunk 10 above)
Comment (DeeJayElly) [reply source/renderer/app/components/widgets/splash/Splash.js]: Done

**Comment by a-rukin** [comment]
> > image
> > is there any overlap? seems like there is something transparent under
> 
> There is some 'text-shadow: 0 -1px 0 rgba(0, 0, 0, 0.25)' on top of active item.
> 
> Should i remove it @a-rukin ?

yes pls

**Comment by a-rukin** [comment]
image
white theme buttons wrong style

**Comment by a-rukin** [comment]
image
styles here copied from ITN/Dark cardano theme, should be from Dark blue

**Comment by a-rukin** [comment]
image
text here in Dark Cardano should be dark (like in ITN)

**Comment by a-rukin** [comment]
image
in light blue and cardano theme this text should be #5e6066 not black, in yellwo and white #2d2d2d

**Comment by a-rukin** [comment]
image
white theme link is invisible

**Comment by a-rukin** [comment]
image

sorry I wasn't very clear about this
hover effect here means:
"x" is always nontransparent, hover affects BG like  in the secondary button style

**Comment by a-rukin** [comment]
image

dark blue theme scroll bar hover effect makes it invisible

**Comment by DeeJayElly** [comment]
> > > image
> > > is there any overlap? seems like there is something transparent under
> > 
> > 
> > There is some 'text-shadow: 0 -1px 0 rgba(0, 0, 0, 0.25)' on top of active item.
> > Should i remove it @a-rukin ?
> 
> yes pls

Fixed

**Comment by DeeJayElly** [comment]
> image
> 
> mini-switch border radius is too big - should be 2px

Fixed

**Comment by DeeJayElly** [comment]
> image
> 
> some padding is still here (less but still) -> need to remove it
> p.s. it can be wrong padding for switch

Fixed

**Comment by DeeJayElly** [comment]
> image
> 
> and
> image
> 
> different font weight
> 
> p.s. filter is corect

Fixed

**Comment by DeeJayElly** [comment]
> image
> 
> white theme buttons wrong style (yellow too)

Fixed

**Comment by DeeJayElly** [comment]
> image
> 
> text here in Dark Cardano should be dark (like in ITN)

Fixed

**Comment by DeeJayElly** [comment]
> image
> 
> styles here copied from ITN/Dark cardano theme, should be from Dark blue

Fixed

**Comment by DeeJayElly** [comment]
> image
> 
> dark blue theme scroll bar hover effect makes it invisible
> 
> image
> 
> light blue - is grey instead of light

Fixed

**Comment by DeeJayElly** [comment]
> image
> 
> in light blue and cardano theme this "selected date" text should be #5e6066 not black, in yellwo and white #2d2d2d

Fixed

**Comment by DeeJayElly** [comment]
> image
> 
> white theme link is invisible
> image
> 
> dark cardano strange too (itn broken in the same way)

Fixed

**Comment by a-rukin** [comment]
@DeeJayElly check this - https://zpl.io/VO5ZQ7k
there you can find secondary style for a theme (rest themes have same too) 
You may notice that normal/hover/press opacity is 0.1/0.05/0.12 for all themes 

image

here opacity goes up (for hover it's 0.3) + for light blue theme text color is blue... why?

**Comment by a-rukin** [comment]
> > image
> > white theme link is invisible
> > image
> > dark cardano strange too (itn broken in the same way)
> 
> Fixed

ITN still bugged

**Comment by DeeJayElly** [comment]
> > > image
> > > white theme link is invisible
> > > image
> > > dark cardano strange too (itn broken in the same way)
> > 
> > 
> > Fixed
> 
> ITN still bugged

Fixed, sorry my commit for this wasn't push so you couldn't see the update. My mistake @a-rukin .

**Comment by DeeJayElly** [comment]
> @DeeJayElly check this - https://zpl.io/VO5ZQ7k
> there you can find secondary style for a theme (rest themes have same too)
> You may notice that normal/hover/press opacity is 0.1/0.05/0.12 for all themes
> 
> image
> 
> here opacity goes up (for hover it's 0.3) + for light blue theme text color is blue... why?

Fixed
